api/specs/web-server/_projects_groups.py

```diff
@@ -0,0 +1,82 @@
+""" Helper script to automatically generate OAS
+
+This OAS are the source of truth
+"""
+
+# pylint: disable=redefined-outer-name
+# pylint: disable=unused-argument
+# pylint: disable=unused-variable
+# pylint: disable=too-many-arguments
+
+
+from _common import assert_handler_signature_against_model
+from fastapi import APIRouter, status
+from models_library.generics import Envelope
+from models_library.projects import ProjectID
+from models_library.users import GroupID
+from simcore_service_webserver._meta import API_VTAG
+from simcore_service_webserver.projects._common_models import ProjectPathParams
+from simcore_service_webserver.projects._groups_api import ProjectGroupGet
+from simcore_service_webserver.projects._groups_handlers import (
+    _ProjectsGroupsBodyParams,
+    _ProjectsGroupsPathParams,
+)
+
+router = APIRouter(
+    prefix=f"/{API_VTAG}",
+    tags=[
+        "projects",
+    ],
+)
+
+
+### Projects groups
+
+
+@router.post(
+    "/projects/{project_id}/groups/{group_id}",
+    response_model=Envelope[ProjectGroupGet],
+    status_code=status.HTTP_201_CREATED,
+)
+async def create_project_group(
+    project_id: ProjectID, group_id: GroupID, body: _ProjectsGroupsBodyParams
+):
+    ...
+
+
+assert_handler_signature_against_model(create_project_group, _ProjectsGroupsPathParams)
```

Comment:
Hmm, I do not see it in the provided example. Maybe you can show me in person so I understand better because it's not yet clear to me.

Reply:
I am closing this review as Frontend changes are merged in the master, so the backend needs to catch up with frontend PR otherwise it is broken. I'll ask you tomorrow about this one.